ci/pipeline.yml

```diff
@@ -153,20 +153,20 @@ jobs:
           run:
             path: src/ci/tasks/restage.sh
         params:
-          <<: *staging-cf
-          CF_APP_NAME: pages-builder-staging
+          <<: *cf
+          CF_APP_NAME: pages-builder-((env))
 
 ############################
 #  RESOURCES
 
 resources:
 
-  - name: src-staging
+  - name: src
     type: git
     icon: github
     source:
       uri: https://github.com/18F/federalist-builder
-      branch: staging
+      branch: ((env))
```

Comment:
this would need to be "main" for production

Reply:
Made a staging and production resource so they can have their own settings https://github.com/18F/federalist-builder/pull/528/files#diff-1d0bbbf28914b9bc860a34ad16f16af0a6996287a6757589f82da847185a8c68R171-R176